src/ui/qml/NetworkSetupWizard.qml

```diff
@@ -93,43 +93,95 @@ ApplicationWindow {
-                width: parent.width
-                text: qsTr("This computer's Internet connection is free of obstacles. I would like to connect directly to the Tor network.")
-                wrapMode: Text.Wrap
-                horizontalAlignment: Qt.AlignHCenter
-            }
+            Column {
+                width: 400
+                spacing: 5
 
-            Button {
-                anchors.horizontalCenter: parent.horizontalCenter
-                text: qsTr("Connect")
-                isDefault: true
-                onClicked: {
-                    // Reset to defaults and proceed to bootstrap page
-                    configPage.reset()
-                    configPage.save()
+                Image {
+                    anchors.horizontalCenter: parent.horizontalCenter
+                    // Ideally we would use the svg here, but I cannot work out a way to get it to
+                    // display as nice as the png does.
+                    source: "../icons/ricochet.png"
                 }
-            }
 
-            Rectangle {
-                height: 1
-                width: parent.width
-                color: palette.mid
+                Label {
+                    width: 400
+                    horizontalAlignment: Qt.AlignHCenter
+                    text: qsTr("Welcome to Ricochet. Before we start, we need to know about your connection to the Internet.")
+                    wrapMode: Text.Wrap
+                }
+                
+                Label {
+                    width:400
+                    horizontalAlignment: Qt.AlignHCenter
+                    text: qsTr("Please try <b>Connect</b> first if you do not require bridges or special network settings.")
+                    wrapMode: Text.Wrap
+                }
             }
 
-            Label {
-                width: parent.width
-                text: qsTr("This computer's Internet connection is censored, filtered, or proxied. I need to configure network settings.")
-                wrapMode: Text.Wrap
-                horizontalAlignment: Qt.AlignHCenter
-            }
+            Row {
+
+                Column {
+                    spacing:5
+
+                    Image {
+                        anchors.horizontalCenter: parent.horizontalCenter
+                        source: "../icons/connection_ok.svg"
+                        width: 200
+                        height: 50
+                    }
+
+                    Button {
+                        anchors.horizontalCenter: parent.horizontalCenter
+                        text: qsTr("Connect")
+                        isDefault: true
+                        onClicked: {
+                            // Reset to defaults and proceed to bootstrap page
+                            configPage.reset()
+                            configPage.save()
+                        }
+                    }
+
+                    Label {
+                        width: 200
```

Comment:
Avoid hardcoding the width; it leads to subtle layout issues and scalability problems. In this case, you could either:

1. Use a RowLayout instead of Row, put `Layout.fillWidth: true` on the Columns
2. Set the Column's width to `(parent.width-1)/2`

In either case, you'd use `width: parent.width` for these items.